src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
If we're allowing the service to control the size of it's container, I think min and max sizing are a valid part of that and are ok here. But ultimately I haven't seen the mockups or anything, so it's hard for me to say if they will ever be used or not. You guys are working this app, you know this better than me :)